File: 2016/finale/interessante/att/interessante.c
#include <stdio.h>
#include <assert.h>

int conta(int N, int K) {
    // Mettete qui il codice della soluzione
    return 42;
}


int main() {
    FILE *fr, *fw;
    int N, K;

    fr = fopen("input.txt", "r");
    fw = fopen("output.txt", "w");
    assert(2 == fscanf(fr, "%d %d", &N, &K));

    fprintf(fw, "%d\n", conta(N, K));
    fclose(fr);
    fclose(fw);
    return 0;
}

File: 2016/finale/interessante/att/interessante.pas
function conta(N, K: longint): longint;
begin
    (* Mettete qui il codice della soluzione *)
    conta := 42;
end;


var
    N, K  : longint;
    fr, fw: text;

begin
    assign(fr, 'input.txt');
    assign(fw, 'output.txt');
    reset(fr);
    rewrite(fw);
    readln(fr, N, K);

    writeln(fw, conta(N, K));
    close(fr);
    close(fw);
end.
